java/src/main/java/com/adaptionsoft/games/trivia/runner/GameRunner.java

```diff
@@ -1,36 +1,36 @@
-
-package com.adaptionsoft.games.trivia.runner;
-import java.util.Random;
-
-import com.adaptionsoft.games.uglytrivia.Game;
-
-
-public class GameRunner {
-
-	private static boolean notAWinner;
-
-	public static void main(String[] args) {
-		Game aGame = new Game();
-		
-		aGame.add("Chet");
-		aGame.add("Pat");
-		aGame.add("Sue");
-		
-		Random rand = new Random(Integer.parseInt(args[0]));
-	
-		do {
-			
-			aGame.roll(rand.nextInt(5) + 1);
-			
-			if (rand.nextInt(9) == 7) {
-				notAWinner = aGame.wrongAnswer();
-			} else {
-				notAWinner = aGame.wasCorrectlyAnswered();
-			}
-			
-			
-			
-		} while (notAWinner);
-		
-	}
-}
+
+package com.adaptionsoft.games.trivia.runner;
+import java.util.Random;
+
+import com.adaptionsoft.games.uglytrivia.Game;
+
+
+public class GameRunner {
+
+	private static boolean notAWinner;
+
+	public static void main(String[] args) {
+		Game aGame = new Game();
+		
+		aGame.addPlayer("Chet");
+		aGame.addPlayer("Pat");
+		aGame.addPlayer("Sue");
+		
+		Random rand = new Random(Integer.parseInt(args[0]));
+	
+		do {
+			
+			aGame.roll(rand.nextInt(5) + 1);
+			
+			if (rand.nextInt(9) == 7) {
+				notAWinner = aGame.wrongAnswer();
+			} else {
+				notAWinner = aGame.wasCorrectlyAnswered();
```

Comment:
notAWinner is set using two methods. Feels like only one method should set it which has state passed into it.